js/controller.js

```diff
@@ -667,12 +667,7 @@ function controller(OO, _, $) {
         // make sure that we don't leave the oo-blur class on the wrong element (since
         // the skin can pick the container rather than the video due to a race condition).
         // Note that this could end up being the video itself, but it shouldn't matter.
-        let videoElementContainer = params.videoElement;
-        if (videoElementContainer && videoElementContainer.length) {
-          [videoElementContainer] = videoElementContainer;
-        }
-        this.state.mainVideoElementContainer = videoElementContainer;
-
+        this.state.mainVideoElementContainer = videoElement[0] || videoElement;
```

Comment:
As I see `this.findMainVideoElement` returns videoElement, not array. So it can be reduced to `this.state.mainVideoElementContainer = videoElement`

Reply:
May be we should use
 `this.state.mainVideoElementContainer = params.videoElement[0] || params.videoElement;` 
here then?
Because otherwise we have `this.state.mainVideoElement = videoElemen` and `this.state.mainVideoElementContainer = videoElement`

Reply:
Yes, that's correct. But you also should check that it's defined